packages/protocol/utils/generate_genesis/taikoL2.ts

```diff
@@ -59,15 +66,22 @@ export async function deployTaikoL2(
 
         // pre-mint ETHs for EtherVault contract
         alloc[contractConfig.address].balance =
-            contractName === "EtherVault"
+            contractName === "ProxiedEtherVault"
```

Comment:
Seems we need to pre-mint L2 ETHs to the `EtherVaultProxy`?

Reply:
done!